catalog/spatial/geocoding/spatial-geocoding-api/src/main/java/org/codice/ddf/spatial/geocoding/GeoEntry.java

```diff
@@ -31,6 +31,8 @@
 
   private final String importLocation;
 
+  private final int gazetteerSort;
```

Comment:
Why is this an int here? A long would seem more consistent with the rest of the code you added

Reply:
The `gazetteerSort` is just an integer value that scores Gazetteer data, which helps sort the suggestions. The only thing that's a long is the population and I use that to determine the score for a piece of data.

Reply:
In that case, would it be more appropriate for getSortPopVal to return an integer instead of a long?